steal.lic

```diff
@@ -20,7 +20,13 @@ class Steal
 
     @stealing_options = get_data('stealing').stealing_options.reject { |target| settings.dont_steal_list.include?(target['id'].to_i) }
     @stealing_bag = settings.stealing_bag
+    @hometown = settings.hometown
+    @stealing_towns = settings.stealing_towns
     @bin_stolen = settings.bin_stolen
+    unless %w[Crossing].include?(@hometown) || !@bin_stolen
```

Comment:
It would  be nice to couple the towns with the existence of a bin room. That way the script doesn't have to be updated for any new bin data